sdk/eventgrid/Microsoft.Azure.Management.EventGrid/tests/Tests/ScenarioTests.cs

```diff
@@ -27,9 +27,7 @@ protected void InitializeClients(MockContext context)
                     if (!m_initialized)
                     {
                         resourceManagementClient = EventGridManagementHelper.GetResourceManagementClient(context, new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK });
-                        resourceManagementClient.BaseUri = new System.Uri("https://eastus2euap.management.azure.com");
                         eventGridManagementClient = EventGridManagementHelper.GetEventGridManagementClient(context, new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK });
-                        eventGridManagementClient.BaseUri = new System.Uri("https://eastus2euap.management.azure.com");
                     }
```

Comment:
Why was this file changed?

Reply:
This is our test code.  In order to expedite the overall process ran the tests against eastus2euap (as this was already deployed). Deployment can take a few days to complete and did not want to be blocked due to that. Once the tests run and confirm its working, there was no need to expect it to be available in eastus2euap always. Hence removed it.